app/controllers/Support.scala

```diff
@@ -0,0 +1,22 @@
+package controllers
+
+import play.api.libs.json.Json
+import play.api.mvc.{Action, Controller}
+import play.api.Logger
+import services.ImageMetadataService
+
+object Support extends Controller with PanDomainAuthActions {
+
+  def imageMetadata(imageUrl: String = "") = AuthAction {
+
```

Comment:
this should probably be an ApiAuthAction so it 419's rather than redirects to google